core/src/main/scala/com/mesosphere/usi/core/logic/MesosEventsLogic.scala

```diff
@@ -128,7 +131,11 @@ private[core] class MesosEventsLogic(mesosCallFactory: MesosCalls) extends Impli
               PodStatus(podId, Map(taskId -> taskStatus))
           }
 
-          SchedulerEvents(stateEvents = List(PodStatusUpdated(podId, Some(newState))))
+          SchedulerEvents(
```

Comment:
we need to test this. But there is another PR from @takirala  introducing test class for this class so I'll not do it now to avoid merge conflicts :)